logstash-integration-jdbc.gemspec

```diff
@@ -36,6 +36,8 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'tzinfo-data'
   # 3.5 limitation is required for jdbc-static loading schedule
   s.add_runtime_dependency 'rufus-scheduler', '< 3.5'
+  s.add_runtime_dependency 'logstash-mixin-ecs_compatibility_support', '~>1.1'
```

Comment:
the latest version is `1.2` which adds support for Elastic stack 8, ECS v8. Potentially, we may need to come back and add ECS v8 to all plugins that use `1.1` before releasing Logstash 8. I would recommend using the latest version now.
`1.2` [advance usage](https://github.com/logstash-plugins/logstash-mixin-ecs_compatibility_support#usage-advanced), [source](https://github.com/logstash-plugins/logstash-mixin-ecs_compatibility_support/pull/5)

Reply:
good point